### DISCUSSION TITLE
Astroneer

### DISCUSSION THREAD

**Discussion by phrygianskittles** [GameServerManagers/LinuxGSM#4651] [Ideas]
I was looking into hosting a dedicated server for a small group of friends for whatever games we wanted to play at the time. I see a bunch of games that we do play, but Astroneer is not included! It's a fun little crafting game, and since it seems to be getting some DLC soon, we were thinking about picking it up again. I didn't see any other posts regarding Astroneer when I search it in all discussions, so I'm wondering if maybe it's just not that well know.
When I asked around on reddit, someone suggest using LinuxGSM since most of the games I asked about seem to be apart of the lineup, but I'm brand new to Linux so I don't really know what I'm doing! haha

I apologize if this question isn't in the right spot. I didn't know if it would go under general, or like questions and answers (since I'm asking about it) but it seems to fit in the ideas category the most!
![images](https://github.com/user-attachments/assets/0a3495e1-82d7-46a4-8f71-c3ffdaa19790)

Thank you!

**Comment by h3o66** [comment]
The astroneer server seems to be windows only , so no native linux support.
Serverid / info on steamdb:
https://steamdb.info/app/728470/
